src/pages/assets/AssetBalancesList.tsx

```diff
@@ -169,45 +171,53 @@ export const AssetBalancesList = observer(function AssetBalancesList() {
 									.toString()}
 								totalFiatValue={totalFiatValue}
 								onDeposit={() => {
-									let modifiedCurrency = currency;
-									if (bal.sourceDenom != '') {
-										modifiedCurrency = {
-											coinDecimals: currency.coinDecimals,
-											coinGeckoId: currency.coinGeckoId,
-											coinImageUrl: currency.coinImageUrl,
-											coinDenom: currency.coinDenom,
-											coinMinimalDenom: '',
-											paths: (currency as IBCCurrency).paths.slice(0, 1),
-											originChainId: bal.sourceChainId,
-											originCurrency: {
+									if (bal.depositUrlOverride) {
+										window.open(bal.depositUrlOverride, '_blank');
+									} else {
+										let modifiedCurrency = currency;
+										if (bal.sourceDenom != '') {
+											modifiedCurrency = {
 												coinDecimals: currency.coinDecimals,
+												coinGeckoId: currency.coinGeckoId,
 												coinImageUrl: currency.coinImageUrl,
 												coinDenom: currency.coinDenom,
-												coinMinimalDenom: bal.sourceDenom,
-											},
-										};
-									}
+												coinMinimalDenom: '',
+												paths: (currency as IBCCurrency).paths.slice(0, 1),
+												originChainId: bal.sourceChainId,
+												originCurrency: {
+													coinDecimals: currency.coinDecimals,
+													coinImageUrl: currency.coinImageUrl,
+													coinDenom: currency.coinDenom,
+													coinMinimalDenom: bal.sourceDenom,
+												},
+											};
+										}
 
-									setDialogState({
-										open: true,
-										counterpartyChainId: bal.chainInfo.chainId,
-										currency: modifiedCurrency as IBCCurrency,
-										sourceChannelId: bal.sourceChannelId,
-										destChannelId: bal.destChannelId,
-										isWithdraw: false,
-										ics20ContractAddress: bal.ics20ContractAddress,
-									});
+										setDialogState({
+											open: true,
+											counterpartyChainId: bal.chainInfo.chainId,
+											currency: modifiedCurrency as IBCCurrency,
+											sourceChannelId: bal.sourceChannelId,
+											destChannelId: bal.destChannelId,
+											isWithdraw: false,
+											ics20ContractAddress: bal.ics20ContractAddress,
+										});
+									}
 								}}
 								onWithdraw={() => {
-									setDialogState({
-										open: true,
-										counterpartyChainId: bal.chainInfo.chainId,
-										currency: currency as IBCCurrency,
-										sourceChannelId: bal.sourceChannelId,
-										destChannelId: bal.destChannelId,
-										isWithdraw: true,
-										ics20ContractAddress: bal.ics20ContractAddress,
-									});
+									if (bal.depositUrlOverride) {
+										window.open(bal.depositUrlOverride, '_blank');
```

Comment:
```suggestion
										window.open(bal.withdrawUrlOverride, '_blank');
```

Reply:
you suggest to change 209 to `window.open(bal.withdrawUrlOverride, ...` but I believe live 208: `if (bal.deposit...` should also be changed to `if (bal.withdraw...`

Reply:
Good catch